finance-api/images/Zettle-card-transaction-flow.svg

```diff
@@ -0,0 +1,3 @@
al">In the merchant's Zettle liquid account, the transaction is recorded.</font></div></div></div></foreignObject><text x="101" y="264" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">In the merchant's Zettle liquid a...</text></switch></g><path d="M 102.66 285 L 102.66 333.63" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 102.66 338.88 L 99.16 331.88 L 102.66 333.63 L 106.16 331.88 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 103 340 L 173 410 L 103 480 L 33 410 Z" fill="#ffffff" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><rect x="53" y="356.25" width="100" height="107.5" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 98px; height: 1px; padding-top: 410px; margin-left: 54px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Arial" data-font-src="https://fonts.googleapis.com/css?family=Arial">Zettle prepares the payout.</font></div></div></div></foreignObject><text x="103" y="414" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Zettle prepares...</text></switch></g><rect x="176" y="370" width="100" height="25" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 98px; height: 1px; padding-top: 383px; margin-left: 177px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Arial">The payout is ready.</font></div></div></div></foreignObject><text x="226" y="386" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">The payout is re...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
If we are to keep this flow chart i feel it's hard to read.
Structuring a flowchart from left to right makes the information easier to read and comprehend.
If you are afraid of it being too wide you can do like in following example
![Screenshot 2021-06-24 at 09 52 48](https://user-images.githubusercontent.com/42367412/123224828-70770100-d4d2-11eb-96e0-c88415f43bc7.png)

Also it's incomplete if we are trying to mimic UML diagram. Diamond is usually a question with "Yes" and "No"  direction. In this diagram we are only describing a happy flow. 
If we are only describing steps then there is no need for diamonds, better to use same shape with different background color to highlight certain steps.

Reply:
![flow_chart_feedback](https://user-images.githubusercontent.com/42367412/123256787-a7104400-d4f1-11eb-849e-9bf183700e43.png)